functions.php

```diff
@@ -34,9 +34,30 @@ function displayGuitars($guitars): string{
             <h1>
                 Brand: ' . $guitar["brand"] . '
                 | Style: ' . $guitar["style"] . '
-                | Neck Wood: ' . $guitar["neckwood"] . '
+                | Year: ' . $guitar["year"] . '
             </h1>
         </article>';
     }
     return $return;
+}
+
+function addGuitarToDatabase(PDO $db, array $guitars): void {
+    $query = $db->prepare("INSERT INTO `guitars` (`brand`, `style`, `year`, `instrument_url`) VALUES (?, ?, ?, ?)");
```

Comment:
please use named placeholders here instead of anonymous ones. if you use the array key names as placeholders you can simplify the execute to just take the array